nextcord/ui/text_input.py

```diff
@@ -0,0 +1,266 @@
+        """:class:`bool`: Whether this component is required to be filled"""
+        if self._underlying.required is not None:
+            return self._underlying.required
+        else:
+            return False
+    
+    @required.setter
+    def required(self, value: Optional[bool]):
+        if value is not None and not isinstance(value, bool):
+            raise TypeError('required must be None or bool')
+        self._underlying.required = value
+    
+    @property
+    def default_value(self) -> Optional[str]:
+        """Optional[:class:`str`]: The value already in the text input when the user open the form."""
+        return self._underlying.value
+    
+    @default_value.setter
+    def default_value(self, value: Optional[str]):
+        if value is not None and not isinstance(value, str):
+            raise TypeError('default_value must be None or str')
+        self._underlying.value = value
+    
+    @property
+    def value(self) -> Optional[str]:
+        """Optional[:class:`str`]: The value sent by the user.
+        This field is updated when an interaction is received.
+        ``TextInput.value`` is ``None`` when no interaction where received.
+        """
+        return self._inputed_value
+    
+    @property
+    def placeholder(self) -> Optional[str]:
+        """Optional[:class:`str`]: The text shown to the user when the text input is empty."""
+        return self._underlying.placeholder
+    
+    @placeholder.setter
+    def placeholder(self, value: Optional[str]):
+        if value is not None and not isinstance(value, str):
+            raise TypeError('placeholder must be None or str')
+        self._underlying.placeholder = value
+    
+    @property
+    def width(self) -> int:
+        return 5
+
+    @classmethod
+    def from_component(cls: Type[T], text_input: TextInputComponent) -> T:
+        return cls(
+            style=text_input.style,
+            custom_id=text_input.custom_id,
+            label=text_input.label,
+            min_length=text_input.min_length,
+            max_length=text_input.max_length,
+            required=text_input.required,
+            value=text_input.value,
+            placeholder=text_input.placeholder,
+            row=None,
+        )
+
+    @property
+    def type(self) -> ComponentType:
+        return self._underlying.type
+
+    def to_component_dict(self):
+        return self._underlying.to_dict()
+
+    def is_dispatchable(self) -> bool:
+        return True
+
+    def refresh_component(self, text_input: TextInputComponent) -> None:
+        self._underlying = text_input
+
+    def refresh_state(self, interaction: Interaction) -> None:
+        data: ComponentInteractionData = interaction.data  # type: ignore
+        for component_data in _walk_all_components(data['components']):
+            if component_data['custom_id'] == self.custom_id:
+                self._inputed_value = component_data['value']
+                return
```

Comment:
```suggestion
                return

```

new line for pep8 ty